riva/clients/nmt/streaming_s2t_client.cc

```diff
@@ -247,23 +247,13 @@ void
 StreamingS2TClient::PostProcessResults(std::shared_ptr<S2TClientCall> call, bool audio_device)
 {
   std::lock_guard<std::mutex> lock(latencies_mutex_);
-  // it is possible we get one response more than the number of requests sent
-  // in the case where files are perfect multiple of chunk size
-  if (call->recv_times.size() != call->send_times.size() &&
-      call->recv_times.size() != call->send_times.size() + 1) {
-    print_latency_stats_ = false;
-  } else {
-    for (uint32_t time_cnt = 0; time_cnt < call->send_times.size(); ++time_cnt) {
-      double lat = std::chrono::duration<double, std::milli>(
-                       call->recv_times[time_cnt] - call->send_times[time_cnt])
-                       .count();
-      if (call->recv_final_flags[time_cnt]) {
-        final_latencies_.push_back(lat);
-      } else {
-        int_latencies_.push_back(lat);
-      }
-      latencies_.push_back(lat);
-    }
+
+  if (simulate_realtime_) {
+    double lat =
+        std::chrono::duration<double, std::milli>(call->recv_times[0] - call->send_times.back())
+            .count();
+    std::cout << "Latency:" << lat << std::endl;
```

Comment:
we are printing latency here as well as from PrintLatencies() function. Is it possible to print only at one place?

Reply:
Here we print individual request latencies, while in PrintLatencies(), in case of multiple iterations the latency values would be average or a percentile value

Reply:
ok, we can move it to VLOG so the print for individual request's latency can be logged if needed.

Reply:
done